content/en/news/releases/1.21.x/announcing-1.21/_index.md

```diff
@@ -82,3 +82,17 @@ This is especially important with the sidecar, because its deployed alongside ev
 workload. Coming in at 25% smaller, the sidecar image can be pulled faster improving pod
 startup times. Additionally, the reduced binary size typically results in a 5MB RAM
 reduction - across many pods, this quickly adds up to cost savings.
+
+### Support for all CNIs in ambient mode
+
+Our new [ambient mode](/blog/2022/introducing-ambient-mesh/) now works across all Kubernetes platforms and CNI implementations, Ambient mode has been tested with GKE, AKS, and EKS and all the CNI implementations they offer, as well as with 3rd-party CNIs like Calico and Cilium, as well as platforms like OpenShift, with solid results. The engineering challenges behind this fix were described in [a recent blog post](/blog/2024/inpod-traffic-redirection-ambient/).
+
+Ambient mode is scheduled to move to Beta in the upcoming Istio 1.22.
```

Comment:
```suggestion
Ambient mode's Layer 4 is scheduled to move to Beta in the upcoming Istio 1.22.
```

Layer 7 will still be alpha as of now.

Reply:
We're going to market "ambient" as beta as a whole, so while technically correct, we'll leave this out for now.

Reply:
Just want to confirm so that we're sending a uniform message, as John said something different https://github.com/istio/istio/pull/49644#issuecomment-1979498018. cc: @howardjohn

Reply:
We discussed this at Steering today and landed with being OK to say "targeted".

Reply:
Just wanted to verify